src/test/04-Stress/10K_array.ck

```diff
@@ -0,0 +1,19 @@
+
+for( 0 => int i; i < foo.cap(); i++ ) {
+ foo[i] => c.freq;
+ .01::T => now;
+}
+<<<"success">>>;
```

Comment:
This test actually fails in tests with:

```shell
> ../chuck ./04-Stress/10K_array.ck
*** Test '10K_array.ck' failed: ***
"success" : (string)
chuck(21307,0x7fff7405e310) malloc: *** error for object 0x100820208: incorrect checksum for freed object - object was probably modified after being freed.
*** set a breakpoint in malloc_error_break to debug
```

I'm wondering if the `--silent` mode is causing this since it does not throw this error when running it without the `--silent` flag.

```shell
$ ../chuck 04-Stress/10K_array.ck
"success" : (string)
$ 

```